diagnose_ceph/troubleshoot_ceph_mon.py

```diff
@@ -0,0 +1,371 @@
+from ConfigParser import ConfigParser
+import os
+import subprocess
+import re
+
+from helpers.exceptions import ConnectionFailedError, TimeoutError
+from helpers.helpers import MyStr
+from troubleshoot_ceph import TroubleshootCeph
+
+
+class MonObject(object):
+    def __init__(self, host, ssh_status, connection=None, admin_socket=None,
```

Comment:
Could you add some docstrings to this?  I'm not sure what admin_socket and connection are supposed to be